### PR TITLE
Productionise

### PR DESCRIPTION
**Pull request by i-like-robots**
Bring this up to speed to be published on NPM.

### CODE REVIEW

**Comment by i-like-robots** [comment]
CC https://github.com/Financial-Times/next-config-vars/pull/869

File: package.json [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "async": "^2.1.5",
-    "split": "^1.0.0"
-  },
+  "name": "@financial-times/n-brightcove-to-uuid",
```
Comment (benbarnett) [inline package.json]: Worth adding a ref to the github repo?

Comment (i-like-robots) [inline package.json]: done

File: .gitignore [hunk 2]
```diff
@@ -1 +1,7 @@
-node_modules
+/node*
+*.env*
+.eslintrc.js
+.editorconfig
```
Comment (benbarnett) [inline .gitignore]: Doesn't look like there's a `.editorconfig` in here btw, prob not a massive deal though?

Comment (i-like-robots) [inline .gitignore]: Config files are auto generated on `make install` so we keep that as the source of truth

Comment (benbarnett) [inline .gitignore]: Sorry, have clearly gone mad. Obviously there isn't one!

**Pull request merged**